Extension/test/unitTests/updowngrade.test.ts

```diff
@@ -35,6 +35,16 @@ suite("UpgradeDowngrade", () => {
                 assert.equal(targetBuild.name, release0);
             });
         });
+        suite("Internal Testing, no Downgrade", () => {
```

Comment:
this new test came here from another branch that is already merged. Please ignore.